src/Models/cell.model.ts

```diff
@@ -8,6 +8,4 @@ const cellSchema = new mongoose.Schema({
     text: String,
 });
 
-const cellModel = mongoose.model<Cell & mongoose.Document>('Cell', cellSchema);
-
-export default cellModel;
+export default cellSchema;
```

Comment:
why are you exporting the schema? export the model, or export both

Reply:
Because I'm not actually defining a model, just using the schema as an embedding in another model.